tests/test_reduction_ops.py

```diff
@@ -535,8 +535,8 @@ def _torch_rms_norm(x, residual, weight, eps):
 
 @pytest.mark.parametrize("shape", REDUCTION_SHAPES)
 @pytest.mark.parametrize("dtype", FLOAT_DTYPES)
-def test_accuracy_softmax(shape, dtype):
-    dim = 1
+@pytest.mark.parametrize("dim", [0, 1])
```

Comment:
DIM_LIST

Reply:
"dim" passed to the function here is an integer.
Do you mean we should add test for non-inner reduction in general?